contentful/content-management/main.js

```diff
@@ -1,5 +1,6 @@
 const path = require("path");
 const fs = require("fs");
+require("dotenv/config");
```

Comment:
How comes this wasn't needed before but is now?

Reply:
before I was exporting the environment variables, it wasn't picking them up from the env file, was my mistake, we need this so that it does pick them up